packages/skill-api/package.json

```diff
@@ -53,11 +53,11 @@
     "preact": "^10.7.2",
     "preact-render-to-string": "^5.2.0",
     "sanitize-html": "^2.7.0",
+    "styled-jsx": "5.1.1",
```

Comment:
this resolved a similar issue where we had THREE pnpm/node_modules nextjs variants